libraries/AP_Scripting/examples/Serial_Dump.lua

```diff
@@ -0,0 +1,44 @@
+-- this script reads data from a serial port and dumps it to a file
+local file_name = 'serial dump.txt'
+local baud_rate = 9600
+
+-- find the serial first (0) scripting serial port instance
+local port = serial:find_serial(0) -- SERIALx_PROTOCOL 28
+
+if not port then
+    error("Could not find Scripting Serial Port")
+    return
+end
+
+-- make a file
+file = io.open(file_name, "w")
+if not file then
+  error("Could not create file " .. file_name)
+end
```

Comment:
```suggestion
file = assert(io.open(file_name, "w"), "Could not create file " .. file_name)
```

Again assert is just a lot shorter

Reply:
fixed